### PR TITLE
Fix `get_coords_level_tile_proxy` description

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: doc/classes/TileSet.xml [hunk 1]
```diff
@@ -113,7 +113,7 @@
 			<argument index="0" name="source_from" type="int" />
 			<argument index="1" name="coords_from" type="Vector2i" />
 			<description>
-				Returns the coodinate-level proxy for the given identifiers. The returned array contains the two proxie's target identifiers (source ID and atlas coords ID).
+				Returns the coordinate-level proxy for the given identifiers. The returned array contains the two proxie's target identifiers (source ID and atlas coords ID).
```
Comment (Calinou) [inline doc/classes/TileSet.xml:116]: We can fix the other typo on the same line while we're at it :slightly_smiling_face: 

```suggestion
				Returns the coordinate-level proxy for the given identifiers. The returned array contains the two proxies' target identifiers (source ID and atlas coordinates ID).
```

Comment (octetdev2) [reply doc/classes/TileSet.xml]: Given the sentence, wouldn't this be "the two proxy target identifiers"?

Comment (akien-mga) [reply doc/classes/TileSet.xml]: Yes, I think you're correct @octetdev2. Another option is "the two target identifiers of the proxy".

Please remember to squash the commits so it's ready to merge.

**Review by Calinou — APPROVED** [review]
Looks good to me, please squash the commits together so we can merge this. See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) for instructions :slightly_smiling_face:

**Comment by octetdev2** [comment]
Squashed, amended and rebased (went with Akien's suggestion).

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**